WordPress/Classes/Services/MediaService.m

```diff
@@ -293,7 +271,7 @@ - (NSError *)translateMediaUploadError:(NSError *)error {
             } break;
         }
         userInfo[NSLocalizedDescriptionKey] = errorMessage;
-        newError = [[NSError alloc] initWithDomain:WordPressComApiErrorDomain code:errorCode userInfo:userInfo];
+        newError = [[NSError alloc] initWithDomain:WordPressComRestApiErrorDomain code:errorCode userInfo:userInfo];
```

Comment:
I get that this was here before, but why is an _XML-RPC_ error on _MediaService_ converted into a `WordPressComRestApiErrorDomain`?